File: Python/kapatici.py
from  os import system
from time import sleep


while True:
    try:
        sor = int(input("1- Otomatik kapatma zamanını ayarla\n2- Otomatik kapatmayı iptal et\n3- Çıkış\n\nSeçim: "))
    except ValueError:
        system("cls")
        print("Lütfen sayı giriniz..")
        sleep(4)
        system("cls")
        continue
    if sor == 1:
        system("cls")
        try:
            sor1 = int(input("Bilgisayarın kaç dakika sonra kapanmasını istiyorsun: "))
        except ValueError:
            system("cls")
            print("Lütfen sayı giriniz..")
            sleep(4)
            system("cls")
            continue
        hesapla = sor1 * 60
        system("shutdown -s -f -t {}".format(hesapla))
        system("cls")
    elif sor == 2:
        system("cls")
        system("shutdown -a")
    elif sor == 3:
        break
    else:
        system("cls")
        print("Lütfen doğru bir seçim yapınız..")
        sleep(4)
        system("cls")

File: C++/kapatici.cpp
#include <iostream>
#include <thread>
#include <chrono>
#include <cstdlib>
using namespace std;

#if __linux__
    const string kapat = "shutdown -h +";
    const string iptal = "shutdown -c";
    const int zort = 1;
#elif _WIN32
    const string kapat = "shutdown /s /t ";
    const string iptal = "shutdown /a";
    const int zort = 60;
#endif

int main(){    
    int menu, secim;
    while(true){
        system("cls||clear");
        cout<<"1- Otomatik kapatma zamanını ayarla\n2- Otomatik kapatmayı iptal et\n3- Çıkış\n\nSeçim: ";
        cin>>menu;
        system("cls||clear");
        if (menu==1){
            cout<<"Bilgisayarın kaç dakika sonra kapanmasını istiyorsun: ";
            cin>>secim;
            system((kapat+to_string(zort*secim)).c_str());
        }
        else if (menu==2){
            system(iptal.c_str());
            cout<<"Otomatik kapatma iptal edildi.."<<endl;
            std::this_thread::sleep_for(std::chrono::seconds(2));
        }
        else if (menu==3){
            break;
        }
        else{
            cout<<"Doğru bir seçim yapınız.."<<endl;
            this_thread::sleep_for(std::chrono::seconds(2));
        }
    }
    return 0;
}